apps/02RockPaperScissors.js

```diff
@@ -7,18 +7,54 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
-
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
+
+  var allowedWords = ['rock', 'paper', 'scissors'];
+  if (allowedWords.indexOf(hand1) < 0 || allowedWords.indexOf(hand2) < 0) {
+    return 'Invalid input';
+  }
```

Comment:
good stuff - input scrubbing is a good practice and also vital for security when you add form inputs to a real-world application